content/docs/plugins/notifications-plugin/_index.md

```diff
@@ -2,48 +2,42 @@
 title: "Notifications Plugin"
 ---
 
-Documentation section of notifications plugin.
+Documentation section of notifications plugin. This plugin is part of Backstage Core.
 
 # Frontend
-The frontend provides both UI and API.  
 The frontend UI is implemented as a side bar item.  
-The following screenshots shows how to use the notifications UI.  
-Whenever a new system notification (update) is detected the frontend pops an alert to the user.  
-
-## Installation and API
-Refer to the plugin's [README](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/notifications/README.md)
+The following screenshots show how to use the notifications UI.  
+Whenever a new notification is detected the frontend pops an alert to the user.  
 
 ## The Frontend UI
 
 ### Notifications page
-![Frontend UI](./fe-screenshot.png)
-
-### New messages icon
-![New messages icon](./new-messages-icon.png)
+![Frontend UI](./sidebar-button-plus-alert.png)
 
-### Filter and sort
-![Filter and sort](./sort-filter.png)
+### Notifications list
+![Notifications list](./list-view.png)
 
-### Updates
-![Updates](./updates.png)
+# Backend
+The backend plugin provides the backend application for reading and writing notifications.
 
-### Update alert
-When a new update notification is received the UI pops an alert
-  
-![Update alert/popup](./system%20alert.png)
+## REST endpoints and OpenAPI
+The plugin does not yet provide an OpenAPI spec. However, we created our own spec for posting notifications. It can be found [here](https://github.com/parodos-dev/serverless-workflows/blob/main/shared/specs/notifications-openapi.yaml).
```

Comment:
You can also generate docs from OpenAPI spec using something like:
```npx --yes @openapitools/openapi-generator-cli@v2.13.1 generate -g asciidoc -i ./openapi.yaml -o docs/index.adoc```

This can help to read/understand the spec.

Reply:
we have just one action. personally, I think that if someone does not understand the openapi syntax then he/she should refrain from using it. it is an opportunity for mistakes.

Reply:
generated adoc. PR is here https://github.com/parodos-dev/serverless-workflows/pull/235